### DISCUSSION TITLE
可以通过配置文件自动支持pwa吗？

### DISCUSSION THREAD

**Discussion by wangshushuo** [reuixiy/hugo-theme-meme#403] [Q&A] [answered]
我看示例的网站时支持的

**Comment by reuixiy** [comment, marked as answer]
https://github.com/reuixiy/hugo-theme-meme/blob/babefc97831b0104270173b07883558638466fe4/config-examples/en/config.toml#L609

开启这项应该就可以，本地开发环境看不到，部署后可以，Chrome 右上角可以安装
